bindings/xmtp_rust_swift/copy_xcframework_to_swift.sh

```diff
@@ -15,13 +15,37 @@ cp -R "XMTPRustSwift.xcframework" "$REPOPATH"
 
 # Need to copy any Swift file in ./include/Generated to $REPOPATH/Sources/XMTPRust/*
 FILES=$(find ./include/Generated -name "*.swift")
+
 # HACK HACK HACK
 # Here's the ultra-hack, we need to inject "import XMTPRustSwift" into the top of the Swift files
 # before copying them over so they'll get the headers we moved to "include/Generated".
+#
+# Moreover, we inject the following lines to allow RustStrings to be interpreted as NSErrors and automatically
+# displayed in the debugger:
+# https://github.com/chinedufn/swift-bridge/issues/150
+#
+# extension RustString: @unchecked Sendable {}
+#
+# extension RustString: LocalizedError {
```

Comment:
🤯 

This is going on my list of things we should make `swift-bridge` issues for. Seems to be actively maintained, but just not a lot of users.

Reply:
I feel like there has to be a better way or I'm misconfiguring something, but this beats their suggestion of just empty `NSError` extension which satisfies the Swift compiler but provides no useful output on error.